cli/run/loadConfigFile.ts

```diff
@@ -39,14 +39,18 @@ export default function loadConfigFile(
 		})
 		.then(({ output: [{ code }] }: RollupOutput) => {
 			// temporarily override require
-			const defaultLoader = require.extensions['.js'];
-			require.extensions['.js'] = (module: NodeModule, filename: string) => {
-				if (filename === configFile) {
-					(module as NodeModuleWithCompile)._compile(code, filename);
-				} else {
-					defaultLoader(module, filename);
-				}
-			};
+			const extensions = ['.js','.mjs'];
```

Comment:
This will work nicely for '.js'  and '.mjs' but not other extensions, and is slightly complicated. How about this: As we KNOW the name of the config file, how about using `path.extname` to extract the extension of the config file and just modify the loader for this extension?